### ISSUE TITLE
Information about this repository.

### ISSUE THREAD

**Issue by ghost** [joaoricardo000/whatsapp-bot-seed#100]
As says in README.MD, this repository is no longer maintained until a new working WhatsApp library is available.
There is another bot framework that seems to work: https://github.com/danielcardeenas/MacBot
![image](https://cloud.githubusercontent.com/assets/11037694/22630929/7ac7e61a-ebd9-11e6-9bb4-a4af6dcb0fa8.png)

**Comment by abdhan** [comment]
I last used this a few months before and I'd like to use again soon. @montemarcaldera I know the issues regarding this framework but just to make sure if it is worth trying to make it usable, can you confirm if it is working for you?

**Comment by ghost** [comment]
I am not using this framework because i don't have a compatible smartphone for using WhatsApp. Currently, Yowsup seems to work partially because the media encryption stuff don't work properly.
I have been looking for an alternative to the official Yowsup repository and [this repository](https://github.com/danielcardeenas/MacBot) has a Yowsup fork that seems to work.

**Comment by ghost** [comment]
@abdhan Make this repository usable is completely worth but isn't easy too.

**Comment by abdhan** [comment]
Okay. I got my number banned today by just running the server. Will try the macBot after I get a new number. Have a good day!

**Comment by joaoricardo000** [comment]
@abdhan and @montemarcaldera, thanks for your guys comments.  
That's a good topic for the #100 issue.  
I gave up fixing the bot after getting banned real quick, after Whatsapp servers stop supporting a protocol version that yowsup had fully implemented. After that, things got unstable really fast.  
If MacBoc yowsup version somehow fixed that, it would be great, although I'm not very optimistic.  
So @abdhan, if you test them, please report back. If they work, we also can.  
Thanks!

**Comment by abdhan** [comment]
Yeah sure! I just need a new number to test it out. A big thanks for your efforts @joaoricardo000 and other contributors. I learnt programming in python for this and I'm still learning. Hope this can be resurrected soon.

**Comment by ghost** [comment]
@abdhan I also learned Python contributing to this project haha

**Comment by abdhan** [comment]
Nice :D

**Comment by Bluscream** [comment]
how to check if youre banned?
